Makefile

```diff
@@ -68,6 +68,8 @@ pypi_upload: ${PACKAGE_DIST} ${PACKAGE_WHEEL}
 
 # Tests
 test_installs:
+	python -m pip uninstall -y torchvision torchaudio
+	python -m pip install torch~=2.2.0 torch_xla[tpu]~=2.2.0 -f https://storage.googleapis.com/libtpu-releases/index.html
```

Comment:
lib versions are hardcoded in a couple of places. Can we have a separate config file / central place to read off such key parameters (that change often)?

Reply:
That was the initial goal of this PR but it pretty hard to have pip right. 
All the attempts I have made did not succeed so I pinned the version(s) this way... 

I don't know if there is a better alternative or an example package that ping these version for pip ?